src/diffusers/models/unet_2d_condition.py

```diff
@@ -34,6 +34,7 @@
     get_down_block,
     get_up_block,
 )
+from .vae import Decoder, Encoder
```

Comment:
The vae should not be important into the unet_2d_condition model, these two classes are unrelated

Reply:
Where do you suggest we add the gradient_checkpointing argument? Should this be done in `autoencoder_kl.py` or `vq_model.py`?

Reply:
Added ability to set gradient checkpointing into `autoencoder_kl.py`